<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>Project Annihilation problem with running</title>
    <link>https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366</link>
    <description>Hey, why the hell I get this error when I&#39;m trying to run it ?

[code]SEVERE: Error starting Hyperion.
java.util.concurrent.ExecutionException: java.io.FileNotFoundException: d:\npcDefinitionsEPICSHIT1337.xml (The device is not ready)
	at java.util.concurrent.FutureTask$Sync.innerGet(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at org.hyperion.util.BlockingExecutorService.waitForPendingTasks(BlockingExecutorService.java:50)
	at org.hyperion.rs2.RS2Server.start(RS2Server.java:74)
	at org.hyperion.Server.main(Server.java:36)
Caused by: java.io.FileNotFoundException: d:\npcDefinitionsEPICSHIT1337.xml (The device is not ready)
	at java.io.FileOutputStream.openAppend(Native Method)
	at java.io.FileOutputStream.&lt;init&gt;(Unknown Source)
	at java.io.FileOutputStream.&lt;init&gt;(Unknown Source)
	at java.io.FileWriter.&lt;init&gt;(Unknown Source)
	at org.hyperion.rs2.model.NPCDefinition.init(NPCDefinition.java:107)
	at org.hyperion.rs2.model.World$4.call(World.java:140)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)[/code]

Could someone say where the problem is ? :/</description>
    
    <lastBuildDate>Tue, 06 Dec 2011 21:26:19 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>Project Annihilation problem with running</title>
        <dc:creator><![CDATA[@The_Outsider The Outsider]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/the_outsider">@The_Outsider</a> wrote:</p>
          <blockquote>
              <p>Navigate to</p>
<pre><code class="lang-auto">package org.hyperion.rs2.model.NPCDefinition;</code></pre>
<p>and find</p>
<pre><code class="lang-auto">d:/npcDefinitionsEPICSHIT1337.xml</code></pre>
<p>and change it to</p>
<pre><code class="lang-auto">data/npcDefinitions.xml</code></pre>
<p>Successful anti-leech was successful.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366/2">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366/2</link>
        <pubDate>Tue, 06 Dec 2011 21:26:19 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-424366-2</guid>
        <source url="https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366.rss">Project Annihilation problem with running</source>
      </item>
      <item>
        <title>Project Annihilation problem with running</title>
        <dc:creator><![CDATA[@addinol addinol]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/addinol">@addinol</a> wrote:</p>
          <blockquote>
              <p>Hey, why the hell I get this error when Iâ€™m trying to run it ?</p>
<p><code>SEVERE: Error starting Hyperion.
java.util.concurrent.ExecutionException: java.io.FileNotFoundException: d:\npcDefinitionsEPICSHIT1337.xml (The device is not ready)
	at java.util.concurrent.FutureTask$Sync.innerGet(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at org.hyperion.util.BlockingExecutorService.waitForPendingTasks(BlockingExecutorService.java:50)
	at org.hyperion.rs2.RS2Server.start(RS2Server.java:74)
	at org.hyperion.Server.main(Server.java:36)
Caused by: java.io.FileNotFoundException: d:\npcDefinitionsEPICSHIT1337.xml (The device is not ready)
	at java.io.FileOutputStream.openAppend(Native Method)
	at java.io.FileOutputStream.&lt;init&gt;(Unknown Source)
	at java.io.FileOutputStream.&lt;init&gt;(Unknown Source)
	at java.io.FileWriter.&lt;init&gt;(Unknown Source)
	at org.hyperion.rs2.model.NPCDefinition.init(NPCDefinition.java:107)
	at org.hyperion.rs2.model.World$4.call(World.java:140)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)</code></p>
<p>Could someone say where the problem is ? <img src="https://forum.moparisthebest.com/images/emoji/twitter/confused.png?v=5" title=":confused:" class="emoji" alt=":confused:"></p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366/1</link>
        <pubDate>Tue, 06 Dec 2011 20:41:25 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-424366-1</guid>
        <source url="https://forum.moparisthebest.com/t/project-annihilation-problem-with-running/424366.rss">Project Annihilation problem with running</source>
      </item>
  </channel>
</rss>
